firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -282,6 +283,25 @@ def get_missing_media(data, import_job_instance):
     return missing_media
 
 
+def delete_unused_media(import_job):
+    """
+    Checks for, and deletes, any media files that were uploaded for the import job but not associated with a media model
+    i.e. Not used in the import.
+    """
+    logger = get_task_logger(__name__)
+
+    try:
+        ImageFile.objects.filter(
+            import_job_id=import_job.id, image__isnull=True
+        ).delete()
+        VideoFile.objects.filter(
+            import_job_id=import_job.id, video__isnull=True
+        ).delete()
+        File.objects.filter(import_job_id=import_job.id, audio__isnull=True).delete()
+    except Exception as e:
+        logger.warning(e)
```

Comment:
Minor: Optional test can be added to simulate an error while deleting media, and verifying the log message.
Optionally, additional text can be added in the warning message too, e.g. `Failed to delete media after job completion`. The error message will contain most of the info, but the extra message could be helpful sometimes.